rfcs/035-marc-856/README.md

```diff
@@ -0,0 +1,359 @@
+
+In the Wellcome catalogue, this is only used for URLs of the form `http://wellcomelibrary.org/item/{bnumber}`, which we don't need to expose in the Catalogue API.
+
+
+## Suggested cataloguing fixes
+
+This table lists the records I found which need some cataloguing fixes:
+
+<table>
+    <tr>
+        <th>bib ID</th>
+        <th>proposed fix</th>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1498281?lang=eng">b1498281x</a></td>
+        <td>The URL in subfield $a should be moved to subfield $u</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1533244?lang=eng">b15332445</a></td>
+        <td>The URL in subfield $a should be moved to subfield $u</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1544605?lang=eng">b15446050</a></td>
+        <td>The URL in subfield $a should be moved to subfield $u</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1550463?lang=eng">b15504633</a></td>
+        <td>The URL in subfield $a should be moved to subfield $u</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb1646418?lang=eng">b16464187</a></td>
+        <td>The URL in subfield $a should be moved to subfield $u</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb2064143?lang=eng">b20641436</a></td>
+        <td>Field 856 doesn't contain any useful information</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb3029755?lang=eng">b30297552</a></td>
+        <td>ind1 = "7" and $2 = "http" should be replaced by ind1 = "4"</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb3029756?lang=eng">b30297564</a></td>
+        <td>The multiple URLs in field 856 $u should be split into multiple instances of field 856</td>
+    </tr>
+    <tr>
+        <td><a href="https://search.wellcomelibrary.org/iii/encore/record/C__Rb3227870?lang=eng">b32278706</a></td>
+        <td>The URL in subfield $z should be moved to subfield $u</td>
+    </tr>
+</table>
+
+
+
+## How we will present field 856 in the Catalogue API
+
+For every instance of field 856, we will create an unidentified Item with a single DigitalLocation.
+The location will have the following fields:
+
+*   `url` = the value of subfield $u.
+
+    If subfield $u is skipped or repeated, we will log a warning and not create the item.
+
+    If the contents of subfield $u doesn't look like a URL, we will log a warning and not create the item.
+
+*   `linkText` = the space-joined contents of subfields $z, $y and $3.
+
+    These will be joined in the original subfield order.
+    If this is an empty string, we will omit the linkText field.
```

Comment:
See discussion in Slack – we don't want this to be arbitrarily long. I'll add some notes from that thread to this PR: https://wellcome.slack.com/archives/C3TQSF63C/p1613990897009600